bin/cli.js

```diff
@@ -30,6 +30,10 @@ const args = yargs
     type: 'boolean',
     describe: 'Generate documentation for methods that have #private notation',
   })
+  .option('generate-contents-table', {
```

Comment:
I'd prefer to name this `contents` in the spirit of `private` option name.

Reply:
Maybe `contents-table`?